File: Easy/Array/3731-Find-Missing-Elements/solution.py
class Solution(object):
    def findMissingElements(self, nums):
        """
        :type nums: List[int]
        :rtype: List[int]
        """
        ans = []
        minnum = min(nums)
        maxnum = max(nums)

        for i in range(minnum, maxnum+1):
            if i not in nums:
                ans.append(i)

        ans.sort()
        return ans

File: Easy/Array/3731-Find-Missing-Elements/solution.cpp
class Solution {
public:
    vector<int> findMissingElements(vector<int>& nums) {
        vector<int> ans;
        int minnum = nums[0];
        int maxnum = nums[0];

        for (int num : nums) {
            if (num < minnum) minnum = num;
            if (num > maxnum) maxnum = num;
        }

        for (int i = minnum; i <= maxnum; i++) {
            bool found = false;
            for (int num : nums) {
                if (num == i) {
                    found = true;
                    break;
                }
            }
            if (!found) {
                ans.push_back(i);
            }
        }

        sort(ans.begin(), ans.end());
        return ans;
    }
};